### ISSUE TITLE
Suggestion: a new GitHub organization for the related projects of PiTest

### ISSUE THREAD

**Issue by paul-hammant** [hcoles/pitest-site#36]
Specifically for collecting all the bits of PiTest together.  

Michal Jedynak's intellij plugin, for example, could move in to the org an still be maintained by him.  Github handles the 302s perfectly, so nothing 404s from older links to technologies.

I wish GitHub had a way of ensuring PiTest itself was first on the list. Maybe that will come another day.

**Comment by hcoles** [comment]
We've had an org for a month or two

https://github.com/pitest

Just need to take the leap of migrating the repos over there.

**Comment by paul-hammant** [comment]
Yeah, the "transfer" wizard at the bottom of the Settings page for the repo, appears daunting, but I've used it a lot, and it is quite pain free.

**Comment by maxgabut** [comment]
@hcoles I think it would be great to migrate the projects to the organisation. I hope you still intend to do it :smiley:
